docs/getstarted/evaluation.md

```diff
@@ -1,36 +1,30 @@
 (get-started-evaluation)=
-# Evaluation
+# Evaluating Your Test Set
```

Comment:
- "Evaluate your test set" means evaluating quality of test set, I would rephrase to "Evaluate *using* your test set"
- Also change in other instances where similar phrase is used.